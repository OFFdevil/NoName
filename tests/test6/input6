main
  int A:=123;
  FUNCa( (struct A));