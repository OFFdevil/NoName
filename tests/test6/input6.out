Function: name= FUNCa didn't call with this parametrs!
